prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-periodics.yaml

```diff
@@ -58,9 +58,9 @@ periodics:
     containers:
     - <<: *config-sync-ci-container
       args:
-      - make
-      - test-e2e-gke-multi-repo-test-group1
-      - 'GCP_CLUSTER=multi-repo-gke-test-group-1-std-stable'
+      #  We're going to replace these two dummy test jobs with real CI jobs.
+      #  Noop them in the interim.
+      - exit 0
```

Comment:
I think you might have to put the exit code as a separate arg

Reply:
DONE. I also reduced the interval from 1h to 1d.